src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -287,8 +287,8 @@ public void run() {
            */
           List<String> masterAddr = j.sentinelGetMasterAddrByName(masterName);  
           if (masterAddr == null || masterAddr.size() != 2) {
-            log.warn("Can not get master addr, master name: {}. Sentinel: {}：{}.",masterName,host,port);
-          }else{
+            log.warn("Can not get master addr, master name: {}. Sentinel: {}: {}.",masterName,host,port);
```

Comment:
why did you add a white space? where is none ascii?

Reply:
The colon is the non ascii character here, GitHub just renders it as such. Before `：` after `: `